File: ejercicio_2.py
num1 = float(input("Ingresa el primer número: "))
num2 = float(input("Ingresa el segundo número: "))

print("1=Suma, 2=Resta, 3=Multiplicación, 4=División")
opcion = int(input("Elige una opción: "))

match opcion:
    case 1:
        print("Resultado:", num1 + num2)
    case 2:
        print("Resultado:", num1 - num2)
    case 3:
        print("Resultado:", num1 * num2)
    case 4:
        if num2 == 0:
            print("Error: no se puede dividir entre 0")
        else:
            print("Resultado:", num1 / num2)
    case _:
        print("Opción no válida")

File: ejercicio_2.cs
using System;

class Calculadora
{
    static void Main()
    {
        Console.Write("Ingresa el primer número: ");
        double num1 = double.Parse(Console.ReadLine());

        Console.Write("Ingresa el segundo número: ");
        double num2 = double.Parse(Console.ReadLine());

        Console.WriteLine("1=Suma, 2=Resta, 3=Multiplicación, 4=División");
        Console.Write("Elige una opción: ");
        int opcion = int.Parse(Console.ReadLine());

        switch (opcion)
        {
            case 1:
                Console.WriteLine("Resultado: " + (num1 + num2));
                break;
            case 2:
                Console.WriteLine("Resultado: " + (num1 - num2));
                break;
            case 3:
                Console.WriteLine("Resultado: " + (num1 * num2));
                break;
            case 4:
                if (num2 == 0)
                    Console.WriteLine("Error: no se puede dividir entre 0");
                else
                    Console.WriteLine("Resultado: " + (num1 / num2));
                break;
            default:
                Console.WriteLine("Opción no válida");
                break;
        }
    }
}